engine/src/cmd/energetic.h

```diff
@@ -100,7 +100,7 @@ class Energetic {
     float warpenergy;    //short fix
     float constrained_charge_to_shields;
     bool sufficient_energy_to_recharge_shields;
-protected:
+public:
```

Comment:
everything prior to this is public, so if we're going to change it then we should just drop it.